### PR TITLE
Setup last layer for GPU images

### PR DESCRIPTION
**Pull request by pacospace**
Related-To: https://github.com/os-climate/aicoe-osc-demo/issues/91
Related-To: https://github.com/os-climate/aicoe-osc-demo/issues/128

This will be the last layer used to build GPU images that have Neural magic for training.

### CODE REVIEW

**Comment by pacospace** [comment]
@harshad16 PTAL :)

File: .s2i/environment [hunk 1]
```diff
@@ -0,0 +1,2 @@
+XDG_CACHE_HOME=/opt/app-root/src/.cache
+THAMOS_RUNTIME_ENVIRONMENT=""
```
Comment (harshad16) [inline .s2i/environment:2]:
```suggestion
THAMOS_RUNTIME_ENVIRONMENT=
```
Tested it , the `""` were causing an issue.
can you please remove it.

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm 
just one minor fix needed

**Comment by harshad16** [comment]
@pacospace the ci check this pr has requires contributor to have signed commits.
As we committed the suggestions they are breaking.
 you might have to squash the commits into one.

**Comment by pacospace** [comment]
> @pacospace the ci check this pr has requires contributor to have signed commits. As we committed the suggestions they are breaking. you might have to squash the commits into one.

Thanks @harshad16, I squashed them.

**Pull request merged**